### PR TITLE
Add instructions how to use esmvaltool from HPC central installations

### PR DESCRIPTION
**Pull request by valeriupredoi**
This is to tell users about the readily available central installations at JASMIN and DRKZ - I filled in the bits for Jasmin, @bjoernbroetz would you be able to get the info in for DKRZ pls mate :beer: 

Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html).

**Tasks**

-   [ ] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [x] This pull request has a descriptive title that can be used in a changelog
-   [ ] Add unit tests
-   [ ] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
-   [ ] If writing a new/modified preprocessor function, please update the [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html)
-   [x] Circle/CI tests pass. Status can be seen below your pull request. If the tests are failing, click the link to find out why.
-   [x] Codacy code quality checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-   [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
-   [ ] If you make backward incompatible changes to the recipe format, make a new pull request in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool) and add the link below

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #issue_number

### CODE REVIEW

**Comment by valeriupredoi** [comment]
@bjoernbroetz @remi-kazeroni can you guys pls have a look at this and add the DKRZ bits pls :beer:

**Comment by valeriupredoi** [comment]
ping @remi-kazeroni - it'd be good to have this feature documented in v2.2 :+1:

**Comment by remi-kazeroni** [comment]
> ping @remi-kazeroni - it'd be good to have this feature documented in v2.2 👍

Good point! I'll ask the people in charge of the central installation at DKRZ what is the status of that 👍

**Comment by valeriupredoi** [comment]
any news here? @remi-kazeroni - Javi and me will get more changes in the release later on in a couple hours so you still have a bit of time, but after that I'd like to merge it, and if no info on DKRZ side we'll do it next (bugfix) release

**Comment by remi-kazeroni** [comment]
> 
> 
> any news here? @remi-kazeroni - Javi and me will get more changes in the release later on in a couple hours so you still have a bit of time, but after that I'd like to merge it, and if no info on DKRZ side we'll do it next (bugfix) release

Sorry I have not heard back from DKRZ about the status of the central installation. I guess this is still ongoing. Hopefully we can include this in the next release.

**Comment by valeriupredoi** [comment]
do you at least know how to module load/activate env to get esmvaltool in path and run stuff on DKRZ? Or is that the info that you wanted to find out from the IT folk

**Comment by remi-kazeroni** [comment]
> 
> 
> do you at least know how to module load/activate env to get esmvaltool in path and run stuff on DKRZ? Or is that the info that you wanted to find out from the IT folk

I'm still waiting for the esmvaltool module to be created

**Comment by valeriupredoi** [comment]
> > do you at least know how to module load/activate env to get esmvaltool in path and run stuff on DKRZ? Or is that the info that you wanted to find out from the IT folk
> 
> I'm still waiting for the esmvaltool module to be created

aha! That changes the status quo - I'll merge only with JASMIN info, keep the issue open and you should tell the IT folk to depeche mode :grin:

**Comment by valeriupredoi** [comment]
blergh this is actually my PR so @jvegasbsc could you pls approve and merge matey :beer:

File: doc/quickstart/install.rst [hunk 1]
```diff
@@ -187,3 +187,14 @@ To install from source for development, follow these instructions.
...
+Pre-installed versions on HPC clusters
+--------------------------------------
+
+You will find the tool available on HPC clusters and there will be no need to install it
+yourself if you are just running diagnostics:
+
+ - CEDA-JASMIN: `esmvaltool` is available on the scientific compute nodes (`sciX.jasmin.ac.uk` where
+   `X = 1, 2,`3, 4, 5`) after login and module loading via `module load esmvaltool`; see the helper page at
+   `CEDA <https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool>`__ ;
+ - DKRZ: Bjoern please
```
Comment (jvegasbsc) [inline doc/quickstart/install.rst:200]: Remove this and we are ok
```suggestion
```

**Review by jvegasbsc — CHANGES_REQUESTED** [review]

**Review by jvegasbsc — APPROVED** [review]

**Pull request merged**